src/main/java/com/es/phoneshop/model/product/ArrayListProductDao.java

```diff
@@ -1,25 +1,48 @@
 package com.es.phoneshop.model.product;
 
+
+import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class ArrayListProductDao implements ProductDao {
+    private List<Product> productList = new ArrayList<>();
+
+    public List<Product> getProductList() {
```

Comment:
This method is intended for internal use only in the test class. It must not be public